packages/fxa-content-server/tests/server/routes/validate-email-domain.js

```diff
@@ -38,43 +23,39 @@ registerSuite('routes/validate-email-domain', {
     'route.process': {
       'responds with {result: "MX"} when there is a MX record': () => {
         const resolveMxStub = sinon.stub().resolves(['mx.abc.xyz']);
-        const dns = {
-          ...ERROR_CODES,
-          promises: {
-            Resolver: function () {
-              this.resolveMx = resolveMxStub;
-            },
-          },
-        };
```

Comment:
We still need to mock `dns` in the test cases that we uses `dns`.

If you run `yarn test-server` from the fxa-content-server directory, you can see which tests are failing.